### DISCUSSION TITLE
Bootstrap Icons

### DISCUSSION THREAD

**Discussion by Fedora-Man** [holger1411/understrap#1635] [Q&A]
What is the best approach to replace Font Awesome with Bootstrap Icons in Understrap?

**Comment by bacoords** [comment]
Typically I comment the Font Awesome import out in the theme.scss file so that the font files and css aren't loaded at all. 

Then you can import Bootstrap icons as an icon font using npm or their CDN link, but for the absolute best speed, I recommend just copy/pasting the actual icon SVG into your theme where appropriate. Inline SVGs are great when compared to loading an entire icon font.


https://icons.getbootstrap.com/#install
